README.md

```diff
@@ -36,10 +36,92 @@
 
 ---
 
+### Quick Links
+
 - ✨ Learn about it on the [docs site](https://ngxs.gitbooks.io/ngxs/)
 - 🚀 See it in action on [Stackblitz](https://stackblitz.com/edit/ngxs-simple)
 - 😎 Checkout the [sample application](integration)
 - 🔧 Scaffolding application using [NGXS Schematics](https://github.com/ngxs/schematics)
 - 📖 Read the blog [posts](https://medium.com/ngxs)
+- ⚡️ Development of [NGXS Labs](https://github.com/ngxs-labs)
 - 📝 Learn about updates from the [changelog](CHANGELOG.md)
 - ❤️ Give back by becoming a [Contributor](docs/community/contributors.md) or a [Sponsor](/docs/community/sponsors.md)
+
+----
+
+### The Goal of NGXS
+
+NGXS tries to make things as simple and accessible as possible. There can be a lot of boilerplate code in state management, thus a main goal of NGXS is to reduce boilerplate allowing you to do more things with less. It is also not necessary to be super familiar with RxJs.
+
+### The Goal of NGXS Labs
+
+The idea with this github organisation is to provide a place for the community to create libraries that augment the main framework with functionality that does not need to be integrated directly into the framework and therefore can evolve through their initial iterations of experimentation without affecting the main @ngxs/store library.
+
+### Getting Started - Local Development
```

Comment:
This is more for getting started with contributing, we should maybe change this title to be:
Getting Started (Contributing)

Maybe we could add a section (maybe just with a link to the relevant guide) for Getting Started wth using the library

Reply:
I was prompted by my friend Minko Gechev, what better to do as they have

https://github.com/angular/angular-cli/blob/master/README.md